common/persistence/telemetry/gowrap_template

```diff
@@ -22,12 +24,37 @@ func new{{upFirst $decorator}} (base {{.Interface.Type}}, tracer trace.Tracer) {
             ctx, span := d.tracer.Start(ctx, "{{ printf "persistence.%s/%s" $.Interface.Name $method.Name }}")
             defer span.End()
 
+            span.SetAttributes(attribute.Key("persistence.store").String("{{ $.Interface.Name }}"))
+            span.SetAttributes(attribute.Key("persistence.method").String("{{ $method.Name }}"))
+
             {{$method.ResultsNames}} = d.{{$.Interface.Name}}.{{$method.Call}}
             {{- if $method.ReturnsError}}
             if err != nil {
               span.RecordError(err)
             }
             {{end}}
+
+            if d.debugMode {
+                {{- if (gt (len $method.Params) 1) }}
+                {{ $request := (index $method.Params 1) }}
+                requestPayload, err := json.MarshalIndent({{ $request.Name }}, "", "    ")
+                if err != nil {
+                    fmt.Println("failed to serialize {{$request.Type}} for OTEL span: " + err.Error())
```

Comment:
This seems good enough as it's only for debugging, ie not for regular use.

Reply:
Both Cass and SQL data store factories has logger and it shouldn't be hard to pass it to constructor of data stores themselves.

Reply:
```suggestion
                    fmt.Println("failed to serialize {{$request.Type}} for OTEL span:", err.Error())
```
If you decide to leave `fmt`.